packages/airnode-node/src/handlers/start-coordinator.test.ts

```diff
@@ -40,6 +39,7 @@ describe('startCoordinator', () => {
         ...chain,
         options: {
           txType,
+          fulfillmentGasLimit: 500000,
```

Comment:
It's minor, but in other areas of our code it's standard practice to use underscores to delineate thousands, eg. `5_000`.
:)

Reply:
Thanks, fixed (I keep forgetting about this) :smile: